gr-digital/python/digital/bindings/docstrings/crc16_async_bb_pydoc_template.h

```diff
@@ -0,0 +1,25 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2021 Cesar Martinez.
+ *
+ * This file is part of GNU Radio
+ *
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ * 
+ */
+#include "pydoc_macros.h"
+#define D(...) DOC(gr, digital, __VA_ARGS__)
+/*
+  This file contains placeholders for docstrings for the Python bindings.
+  Do not edit! These were automatically extracted during the binding process
```

Comment:
if generated, should these be controlled?

Reply:
Yes. This is how the pybind11 bindings are handled. Probably others can give good reasons why these are in the VCS.